enzyme/benchmarks/ReverseMode/adbench/gmm.h

```diff
@@ -168,8 +168,8 @@ int main(const int argc, const char* argv[]) {
     std::vector<std::string> paths;// = { "1k/gmm_d10_K100.txt" };
 
     getTests(paths, "data/1k", "1k/");
-    getTests(paths, "data/2.5k", "2.5k/");
-    getTests(paths, "data/10k", "10k/");
+    //getTests(paths, "data/2.5k", "2.5k/");
+    //getTests(paths, "data/10k", "10k/");
```

Comment:
Let's try to get this merged.

Here, can you use an environmental or command line variable to determine whether to run large tests or not?